tests/lib/ocpp/v201/test_composite_schedule.cpp

```diff
@@ -813,4 +813,27 @@ TEST_F(ChargepointTestFixtureV201, K08_CalculateCompositeSchedule_ExternalOverri
     ASSERT_EQ(actual, expected);
 }
 
+TEST_F(ChargepointTestFixtureV201, K08_CalculateCompositeSchedule_NoProfiles) {
```

Comment:
Could the test name mention the expected outcome results?

Reply:
Renamed to `K08_CalculateCompositeSchedule_NoProfilesAppliesNoLimits`